recipe-client-addon/lib/PreferenceExperiments.jsm

```diff
@@ -61,105 +61,21 @@ XPCOMUtils.defineLazyModuleGetter(this, "LogManager", "resource://shield-recipe-
 XPCOMUtils.defineLazyModuleGetter(this, "Preferences", "resource://gre/modules/Preferences.jsm");
 XPCOMUtils.defineLazyModuleGetter(this, "TelemetryEnvironment", "resource://gre/modules/TelemetryEnvironment.jsm");
 
-this.EXPORTED_SYMBOLS = ["PreferenceExperiments"];
-
-const EXPERIMENT_FILE = "shield-preference-experiments.json";
-
-const PREFERENCE_TYPE_MAP = {
-  boolean: Services.prefs.PREF_BOOL,
-  string: Services.prefs.PREF_STRING,
-  integer: Services.prefs.PREF_INT,
-};
-
-const DefaultPreferences = new Preferences({defaultBranch: true});
-
-/**
- * Enum storing Preference modules for each type of preference branch.
- * @enum {Object}
- */
-const PreferenceBranchType = {
-  user: Preferences,
-  default: DefaultPreferences,
-};
+XPCOMUtils.defineLazyModuleGetter(this, "PreferenceManagement", "resource://shield-recipe-client/lib/PreferenceManagement.jsm");
```

Comment:
Well, if we need to check this stuff on startup (to flip prefs) it will never be lazy, I think, so we should probably just make it more obvious that when someone imports this module, PreferenceManagement is imported sync (Cu.import() rather than defineLazyModuleGetter).